pangeo_forge_recipes/recipes/xarray_zarr.py

```diff
@@ -621,9 +621,25 @@ def store_chunk(
                 zarr_array[zarr_region] = data
 
 
-def finalize_target(target: CacheFSSpecTarget, consolidate_zarr: bool) -> None:
+def finalize_target(
+    target: CacheFSSpecTarget,
+    consolidate_zarr: bool,
+    consolidate_dimension_coordinates: bool = True,
+) -> None:
     if target is None:
         raise ValueError("target has not been set.")
+
+    if consolidate_dimension_coordinates:
+        logger.info("Consolidating dimension coordinate arrays")
+        target_mapper = target.get_mapper()
+        ds = xr.open_zarr(target_mapper)  # Probably a better way to get the dimension coords?
+        group = zarr.open(target_mapper)
+        for dim in ds.dims:
+            attrs = dict(group[dim].attrs)
+            data = group[dim][:]
+            group[dim] = data
+            group[dim].attrs.update(attrs)
```

Comment:
Why are you updating the attrs here? It doesn't look like they could have changed at all.